Simplified/NYPLNetworkQueue.swift

```diff
@@ -92,31 +108,29 @@ final class NetworkQueue: NSObject {
       Log.error(#file, "SQLite Error: Could not insert or update row")
     }
   }
-  
-  class func retryQueue()
+
+  private func retryQueue()
   {
-    guard let db = startDatabaseConnection() else { return }
-    
-    do {
-      for row in try db.prepare(sqlTable) {
-        self.retry(db: db, requestRow: row)
+    serialQueue.async {
```

Comment:
_Extremely_ minor, but you use `serialQueue.async` here and `self.serialQueue.async` below. I'd prefer the latter in both places but consistency would be nice either way.